CONTRIBUTING.md

```diff
@@ -1,156 +1,154 @@
+
+1. If you do not have a [GitHub][github] account, you can [send us comments by
+   email][contact]. However, we will be able to respond more quickly if you use
+   one of the other methods described below.
+
+2. If you have a [GitHub][github] account, or are willing to [create
+   one][github-join], but do not know how to use Git, you can report problems
+   or suggest improvements by [creating an issue][issues]. This allows us to
+   assign the item to someone and to respond to it in a threaded discussion.
+
+3. If you are comfortable with Git, and would like to add or change material,
+   you can submit a pull request (PR). Instructions for doing this are
+   [included below](#using-github).
+
+Note: if you want to build the website locally, please refer to [The Workbench
+documentation][template-doc].
+
+### Where to Contribute
+
+1. If you wish to change this lesson, add issues and pull requests here.
+2. If you wish to change the template used for workshop websites, please refer
+   to [The Workbench documentation][template-doc].
+
+
+### What to Contribute (General)
+
+There are many ways to contribute, from writing new exercises and improving
+existing ones to updating or filling in the documentation and submitting [bug
+reports][issues] about things that do not work, are not clear, or are missing.
+If you are looking for ideas, please see [the list of issues for this
+repository][repo], or the issues for [Data Carpentry][dc-issues], [Library
+Carpentry][lc-issues], and [Software Carpentry][swc-issues] projects.
+
+Comments on issues and reviews of pull requests are just as welcome: we are
+smarter together than we are on our own. **Reviews from novices and newcomers
+are particularly valuable**: it's easy for people who have been using these
+lessons for a while to forget how impenetrable some of this material can be, so
+fresh eyes are always welcome.
+
+### What to Contribute (This Lesson)
+
+The curriculum is relatively new and under active development/refinement,
+so any contribution will be welcome and probably very helpful.
+Ideas for how the existing content could be improved or updated,
+and/or errors that need to be corrected will be particularly useful,
+but we are certainly open to suggestions for new content and all other kinds of feedback.
+The curriculum currently lacks figures and other visual aids to teaching,
+so these will be particularly appreciated.
+Comments on existing issues and reviews of pull requests are similarly welcome,
+whether you are a certified Lesson Developer Trainer for the program or not.
+
+If you plan to submit a pull request, please open an issue
+(or comment on an existing thread) first to ensure that effort is not duplicated
+or spent making a change that will not be accepted by the Maintainers.
+
+### What *Not* to Contribute (General)
+
+Our lessons already contain more material than we can cover in a typical
+workshop, so we are usually *not* looking for more concepts or tools to add to
```

Comment:
```suggestion
workshop, so we are usually *not* looking for more content to add to
```

Reply:
I am going to leave this in, as it is part of the "general" section and relates to all Carpentries lessons. You could argue that "adding tools" to a lesson is a subcategory of "adding content" but I think the redundancy is worthwhile to help make the point e.g. to someone who wants to suggest adding another command to a Shell lesson or similar.